coral-spark/src/test/java/com/linkedin/coral/spark/CoralSparkTest.java

```diff
@@ -750,7 +750,7 @@ public void testNestedFieldProjectionWithSameNameAlias() {
     String sourceSql = "SELECT complex.s.name as name FROM default.complex";
     String expandedSql = getCoralSparkTranslatedSqlWithAliasFromCoralSchema(sourceSql);
 
-    String targetSql = "SELECT s.name name\n" + "FROM default.complex";
+    String targetSql = "SELECT complex.s.name name\n" + "FROM default.complex";
```

Comment:
Not sure if this kind of change works on Spark, let's test it on gateway Spark shell and update that in the PR description (same for Trino).

Reply:
Integ tested with both spark & trino. This change is a non-issue